wiki/fabrication/manufacturing-technique-selection-guide.md

```diff
@@ -0,0 +1,29 @@
+---
+Manufacturing Technique Selection Guide
+---
+
+## Laser Cutting Guide
+This link is to both an introductory and advanced topic laser cutting guide that seeks to answers all your laser cutting questions. The guide starts off by giving a brief history of lasers to the common laser cutting machines of today. Multiple diagrams explaining how a typical laser cutter works are provided that really help the reader understand what is physically going on when their design is cut. The guide also provides an in-depth set of guidelines for creating both laser cutter specific parts, like living hinges, as well as general rules for cutting basic mechanical devices like intermeshing gears. 
+
+## See Also:
+- https://www.sculpteo.com/media/ebook/Sculpteo_ultimate_guide_laser_cutting.pdf
```

Comment:
Should be something like:
`[Sculpteo's Ultimate Guid to Laser Cutting](https://www.sculpteo.com/media/ebook/Sculpteo_ultimate_guide_laser_cutting.pdf)`